beta-src/src/enums/Vote.ts

```diff
@@ -0,0 +1,7 @@
+enum Vote {
+  DRAW = "draw",
+  PAUSE = "pause",
+  CANCEL = "cancel",
+}
+
+export default Vote;
```

Comment:
i think greg needs this in his pr or it's there already not sure.

Reply:
Yea, we'll need to converge these once both PRs are in